spec/services/record_declarations/base_spec.rb

```diff
@@ -50,7 +126,7 @@ def matches_lead_provider?
     end
 
     it "does not have errors on milestone_date" do
-      expect { subject.call }.not_to raise_error(ActionController::ParameterMissing)
```

Comment:
This was creating a false positive as the warned by RSpec.
The error was the example model was missing `def self.declaration_model`